send/ad_mu

```diff
@@ -469,33 +476,52 @@ process_licenses_groups();
+ldap_log($service_name, "Group grace period set: " . $counter_group_grace_period_set . " entries.");
+ldap_log($service_name, "Group grace period failed to set: " . $counter_group_grace_period_not_set . " entries.");
+ldap_log($service_name, "Group set as removed: " . $counter_group_removed . " entries.");
+ldap_log($service_name, "Group failed to set as removed: " . $counter_group_not_removed . " entries.");
+ldap_log($service_name, "Group attributes updated: " . $counter_group_attributes_updated . " entries.");
+ldap_log($service_name, "Group attributes failed to update: " . $counter_group_attributes_not_updated . " entries.");
+ldap_log($service_name, "Group members updated: " . $counter_group_members_updated . " entries.");
+ldap_log($service_name, "Group members updated with errors: " . $counter_group_members_updated_with_errors . " entries.");
+ldap_log($service_name, "Group members failed to update: " . $counter_group_members_not_updated . " entries.");
 
 # print results
-print "Added: " . $counter_add . " entries.\n";
-print "Updated: " . $counter_updated . " entries.\n";
-print "Disabled: " . $counter_disabled . " entries.\n";
-print "Failed: " . $counter_fail. " entries.\n";
+print "User added: " . $counter_add . " entries.\n";
+print "User updated: " . $counter_updated . " entries.\n";
+print "User disabled: " . $counter_disabled . " entries.\n";
+print "User failed: " . $counter_fail. " entries.\n";
 print "OU added: " . $counter_add_ous . " entries.\n";
 print "OU failed: " . $counter_fail_ous . " entries.\n";
-print "Group added: " . $counter_group_add . " entries.\n";
-print "Group removed: " . $counter_group_remove . " entries.\n";
-print "Group updated without errors: " . $counter_group_updated . " entries.\n";
-print "Group updated with errors: " . $counter_group_updated_with_errors . " entries.\n";
-print "Group failed: " . $counter_group_failed . " entries.\n";
+print "Group added (without members): " . $counter_group_added . " entries.\n";
+print "Group failed to add: " . $counter_group_not_added . " entries.\n";
+print "Group failed to empty during removal: " . $counter_group_not_emptied . " entries.\n";
+print "Group grace period set: " . $counter_group_grace_period_set . " entries.\n";
+print "Group grace period failed to set: " . $counter_group_grace_period_not_set . " entries.\n";
+print "Group set as removed: " . $counter_group_removed . " entries.\n";
+print "Group failed to set as removed: " . $counter_group_not_removed . " entries.\n";
+print "Group updated (attributes): " . $counter_group_attributes_updated . " entries.\n";
+print "Group failed to update (attributes): " . $counter_group_attributes_not_updated . " entries.\n";
+print "Group updated (members): " . $counter_group_members_updated . " entries.\n";
+print "Group updated with errors (members): " . $counter_group_members_updated_with_errors . " entries.\n";
+print "Group failed to update (members): " . $counter_group_members_not_updated . " entries.\n";
```

Comment:
Similar changes of output as in ad_group_mu_ucn is needed here too.

Reply:
Done